assets/js/services/lesson_builder.js

```diff
@@ -0,0 +1,13 @@
+app.factory("LessonBuilder", ["Lesson", function(Lesson) {
+
+  var editLesson = function(body, lesssonId) {
+    var lesson = Lesson.get({id: lessonId}, function(data) {
```

Comment:
How else should we update? This is the way that the learn-app shows how to update...